<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>412</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>412</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>412</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #F0EFFF;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QCustomPlot" name="graphics" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>71</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCustomPlot{
	border: 1px transparent;
	color: #4F4C79;
	border-radius: 8px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>19</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 20pt &quot;.AppleSystemUIFont&quot;;
	color: #4E4B78;
	background-color: #F2F1FF;
	border: 1px transparent;
	border-radius: 8px;
}</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>expression</string>
   </property>
   <property name="maxLength">
    <number>256</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="plot_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>279</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border: 1px transparent;
	color: #4F4C79;
	border-radius: 8px;
	background-color: #E8E7FC;
	font-size: 14px;
}

QPushButton:hover {
	background-color: #F5BCB7;
}

QPushButton:pressed {
	background-color: #BEBBF6;
}</string>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_plot">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border: 1px transparent;
	color: #4F4C79;
	border-radius: 8px;
	background-color: #E8E7FC;
	font-size: 14px;
}

QPushButton:hover {
	background-color: #F5BCB7;
}

QPushButton:pressed {
	background-color: #BEBBF6;
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>120</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="x_Min">
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #4F4C79;
	background-color: #F0EFFF;
	border: 1px transparent;
	border-radius: 18px;
}</string>
      </property>
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QDoubleSpinBox" name="x_min">
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox::pane {
	border: 1px transparent;
	background-color: #F0EFFF;
	color: #E8E7FC;
	font-size: 10px;
}

QDoubleSpinBox{
	border: 1px transparent;
	color: #4F4C79;
	border-radius: 8px;
	background-color: #E8E7FC;
}
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="x_max">
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox::pane {
	border: 1px transparent;
	background-color: #F0EFFF;
	color: #E8E7FC;
	font-size: 10px;
}

QDoubleSpinBox{
	border: 1px transparent;
	color: #4F4C79;
	border-radius: 8px;
	background-color: #E8E7FC;
}
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="x_Max">
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #4F4C79;
	background-color: #F0EFFF;
	border: 1px transparent;
	border-radius: 18px;
}</string>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
